src/sbt-test/dumpLicenseReport/custom-report/test

```diff
@@ -1,4 +1,5 @@
 > dumpLicenseReport
 $ exists target/license-reports/lreport.md
 > check
-$ exists target/license-reports/test-config.html
\ No newline at end of file
+$ exists target/license-reports/test-config.html
```

Comment:
Oh, never mind: I was mixing up the annotations from the diff with the actual file contents.